src/Adapter/MailTemplate/MailTemplateTwigRenderer.php

```diff
@@ -67,12 +70,14 @@ class MailTemplateTwigRenderer implements MailTemplateRendererInterface
     public function __construct(
         Environment $twig,
         LayoutVariablesBuilderInterface $variablesBuilder,
-        HookDispatcherInterface $hookDispatcher
+        HookDispatcherInterface $hookDispatcher,
+        bool $hasGiftWrapping
```

Comment:
Can you update the phpdoc?

Reply:
No sorry, I was talking about lines 64-66:
```php
/**
 * @param Environment $twig
 * @param LayoutVariablesBuilderInterface $variablesBuilder
 * @param HookDispatcherInterface $hookDispatcher
```

Reply:
Done 😉